translate/rus.json

```diff
@@ -168,6 +169,7 @@
 "Final FEN":"окончательное FEN",
 "final":"финал",
 "first":"первый",
+"first~2":"начало",  
```

Comment:
Not sure if this word and 'last' one as well used elsewhere, so decided to go safe than sorry. These translations should be used here:
![TCEC rus](https://user-images.githubusercontent.com/77082125/105915564-e9b41a00-6040-11eb-96c1-ea89a2f793b8.png)

Reply:
I checked the code, `first` only appears there, so you can rename the original `first` to `начало` and remove the `first~2`.

Reply:
And for 'last'? Same story?

Reply:
last is used here:
- show ply
- audio moves
- audio from

Reply:
Actually, I don't think начало and конец are needed, первый and последний are OK, because show ply -> which ply? "first ply", "last ply".

Reply:
Where is 'audio from' located? Can't find it from first glance.

Reply:
Seems reasonable @octopoulos, but my Russian loc has a slightly different. Mine asks: 'What point of PV you want to show me?' And the answers are: 'the start', 'the end', and 'the divergence point'. I hope you understood, since it is a bit different, and this is the way I see it.

Reply:
@octopoulos I think I found the way around this problem. Will change the PR now.